src/v2/transaction.rs

```diff
@@ -14,6 +18,14 @@ use num_bigint::BigInt;
 #[cfg(feature = "serde")]
 use serde::{Deserialize, Serialize};
 
+use super::{
```

Comment:
The same applies here, could you move everything from `crate::v2` to `super`, and remove the empty line